doc/api/cluster.markdown

```diff
@@ -402,21 +402,21 @@ See: [Child Process module](
 child_process.html#child_process_child_process_fork_modulepath_args_options)
 
 Note that workers will call `process.exit(0)` if the `'disconnect'` event occurs
-on `process` and `.suicide` is not `true`. This protects against accidental
+on `process` and `.voluntaryExit` is not `true`. This protects against accidental
 disconnection.
 
-### worker.suicide
+### worker.voluntaryExit
```

Comment:
At least for the short term, we should probably not that this used to be called `suicide`, just so people don't get confused.

(If we are landing in v5 or v4)